### PR TITLE
Detail page redesigned with grid CSS

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Completed redesign the detail page with grid CSS

## Screenshot
**Desktop**
Captura de Pantalla 2020-11-09 a la(s) 12 12 08 p  m
**Mobile**
Captura de Pantalla 2020-11-09 a la(s) 12 11 53 p  m

## Trello Card
https://trello.com/c/pt9mL9Tv/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

**Review by mcallegari10 — APPROVED** [review]

File: maquetacion/src/scss/_detail.scss [hunk 1]
```diff
@@ -41,27 +54,35 @@ $badge-width: 90px;
     font-family: $primary-font;
     font-size: 24px;
     font-weight: 700;
+    grid-area: genre;
+    margin: 0 auto 25px auto;
 
     @media screen and (max-width: $lg) {
+      border-bottom: 4px solid $wolox-green;
       font-size: 16px;
+      margin: 15px auto 10px auto;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss]:
```suggestion
      margin: 15px auto 10px;
```

File: maquetacion/src/scss/_detail.scss [hunk 2]
```diff
@@ -41,27 +54,35 @@ $badge-width: 90px;
     font-family: $primary-font;
     font-size: 24px;
     font-weight: 700;
+    grid-area: genre;
+    margin: 0 auto 25px auto;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss]:
```suggestion
    margin: 0 auto 25px;
```

File: maquetacion/src/scss/_detail.scss [hunk 3]
```diff
@@ -25,13 +25,26 @@ $badge-width: 90px;
 .detail {
   background-color: $light;
   box-shadow: 4px 4px 10px 0 $half-black;
-  display: flex;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  grid-template-areas:
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss:29]: sort by name

File: maquetacion/src/scss/_detail.scss [hunk 4]
```diff
@@ -25,13 +25,26 @@ $badge-width: 90px;
...
+  margin-bottom: 30px;
   max-width: 900px;
   padding: 24px;
   width: 100%;
 
   @media screen and (max-width: $lg) {
-    flex-direction: column;
+    grid-template-columns: 1fr;
+    grid-template-rows: 60px 300px 60px 1fr;
+    grid-template-areas:
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss:41]: sort by name

File: maquetacion/src/scss/_detail.scss [hunk 5]
```diff
@@ -41,27 +54,35 @@ $badge-width: 90px;
...
+      text-align: center;
+      width: 100%;
     }
   }
 
   &__book-title {
     font-size: 32px;
     font-weight: 700;
+    grid-area: title;
+    margin: 10px auto 0 auto;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss]:
```suggestion
    margin: 10px auto 0;
```

File: maquetacion/src/scss/_detail.scss [hunk 6]
```diff
@@ -41,27 +54,35 @@ $badge-width: 90px;
...
 
     @media screen and (max-width: $lg) {
       font-size: 24px;
+      margin: 10px auto 25px auto;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss]:
```suggestion
      margin: 10px auto 25px;
```

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

File: maquetacion/src/scss/_detail.scss (hunk 3 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:29]: Sorted

File: maquetacion/src/scss/_detail.scss (hunk 4 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:41]: Sorted

**Review by seromarin — COMMENTED** [review]
Fixed all comments

**Review by idcuadrowolox — APPROVED** [review]

**Pull request merged**